d2l-scroll-wrapper.html

```diff
@@ -349,9 +349,18 @@
 					showActions: Boolean
 				},
 
-				listeners: {
-					'iron-resize': 'checkScrollbar',
-					'wrapper.scroll': 'checkScrollThresholds'
+				attached: function() {
+					// Defer the offsetWidth/scrollWidth calculations until after the page has rendered
+					Polymer.RenderStatus.afterNextRender(this, function() {
+						this.listen(this, 'iron-resize', 'checkScrollbar');
+						this.listen(this.$.wrapper, 'scroll', 'checkScrollThresholds');
```

Comment:
The `iron-resize` event is fired when the element is attached. So yes.

I `wrapper.scroll` I don't think would be, but it's good for consistency.